node/node_test.go

```diff
@@ -0,0 +1,58 @@
+package node
+
+import (
+	"net/http"
+	"testing"
+	"time"
+
+	"go.sia.tech/renterd/internal/consensus"
+)
+
+type noopHandler struct{}
+
+func (h *noopHandler) ServeHTTP(http.ResponseWriter, *http.Request) {
```

Comment:
since `http.Handler` is an interface can't we just pass `nil` and check for `nil`

Reply:
Nice thing is that this way we can verify that `Node` registers the handler correctly even if it doesn't do anything.